### PR TITLE
fix radians to direction

### PR DESCRIPTION
**Pull request by caplanmaor**

### CODE REVIEW

File: NadavAI/visualization/visualization.cpp [hunk 1]
```diff
@@ -22,18 +21,15 @@ Agent::Agent(const Entity &entity) {
 
 Direction Agent::radiansToDirection(Radian angle) {
     auto degrees = angle.toDegrees();
-    if ((degrees >= 0 && degrees <= 45) || (degrees >= 315 && degrees <= 360)) {
+    if (degrees >= 225 && degrees <= 315) {
```
Comment (nodeav) [inline NadavAI/visualization/visualization.cpp]: optionally don't convert to degrees and use `angle.between(from, to)` which has tests

Comment (caplanmaor) [reply NadavAI/visualization/visualization.cpp]: fixed gracias maestro

File: NadavAI/utils.h [hunk 2]
```diff
@@ -39,6 +39,10 @@ class Radian {
         return value_ * 180.0 / M_PI;
     }
 
+    Radian toRadian(float dergees) const {
```
Comment (nodeav) [inline NadavAI/utils.h]: make this static, and call it `fromDegrees` (so: `Radian.fromDegrees(315.)`)

Comment (nodeav) [reply NadavAI/utils.h]: (static because it doesn't access `this` and it doesn't make any sense for it to)

**Review by nodeav — COMMENTED** [review]
looks ok except for my comment,
and additionally please modify the bad commit (you can use an interactive rebase with the `fixup` command; you have a simple guide in vim/nano when you run `git rebase -i`

**Comment by caplanmaor** [comment]
> looks ok except for my comment, and additionally please modify the bad commit (you can use an interactive rebase with the `fixup` command; you have a simple guide in vim/nano when you run `git rebase -i`

@nodeav fixed thanks 👍🏼

**Comment by nodeav** [comment]
@caplanmaor abale you've accidentally committed my commits again, guessing your didn't `pull` before rebasing (or alternatively `git fetch` and then `git rebase origin/main`)

**Review by nodeav — APPROVED** [review]

**Pull request merged**